src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicAnalysisDataReader.cs

```diff
@@ -174,6 +174,11 @@ private static int ReadReference(ref BlobReader reader, bool smallRefSize)
 
         public ImmutableArray<LinePositionSpan> GetSpans(BlobHandle handle)
         {
+            if (handle.IsNil)
```

Comment:
This happens if a method has no spans. Without this check the method crashes later.